xwiki-commons-core/xwiki-commons-cache/xwiki-commons-cache-infinispan/pom.xml

```diff
@@ -50,6 +50,11 @@
       <groupId>org.infinispan</groupId>
       <artifactId>infinispan-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.microprofile.metrics</groupId>
+      <artifactId>microprofile-metrics-api</artifactId>
+      <version>3.0</version>
+    </dependency>
```

Comment:
Should this be optional?

Reply:
I'd also add some comment explaining this dep

Reply:
Thanks Vincent for the review. I realize that the dependency should actually be added only to the modules needing such a JMX access, so this is the wrong place I guess, apologies. Let me investigate a bit further and probably I will close the PR and the related issue.

Reply:
No I think this is the right place. Just missing a comment explaining why we need this dependency now.

Reply:
I agree that it would be nice to be able to use JMX to access infinispan stats by default in XS. I'm not sure the location is right since IMO it could also be in the minimal dependencies module in platform (for the base flavor).

Is this a regression due to an upgrade of infinispan? I'm positive it was working before. If it's a regression I would mark XCOMMONS-2188 as such since I believe we can consider JMX access as base feature of XS.

Reply:
Yes it's a regression due to an upgrade of Infinispan which introduced this dependency in the 10.x lifecycle, I mentioned the related Infinispan commits / issues in XCOMMONS-2188, which Thomas marked as a regression indeed in the meantime.